scene/gui/color_picker.h

```diff
@@ -79,6 +178,13 @@ class ColorPicker : public BoxContainer {
 	static Ref<Shader> circle_ok_color_shader;
 	static List<Color> preset_cache;
 
+	int current_sliders_count = 4;
+
+	ColorMode *selected_mode;
+	Vector<ColorMode *> modes;
+	const int HSV_MODE_INDEX = 1;
+	const int OKHSL_MODE_INDEX = 3;
```

Comment:
This shouldn't exist.

Reply:
There are checks for color mode being HSV or OKHSL in `_w_input`.

Reply:
Yeah but this is more related to shapes than color modes. Maybe it's possible to handle it more nicely.